src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -834,7 +832,7 @@ std::optional<std::string> CascadiaSettings::_ReadUserSettings()
         // GH#5186 - We moved from profiles.json to settings.json; we want to
         // migrate any file we find. We're using MoveFile in case their settings.json
         // is a symbolic link.
-        auto pathToLegacySettingsFile{ pathToSettingsFile };
+        std::filesystem::path pathToLegacySettingsFile{ pathToSettingsFile.c_str() };

```

Comment:
```suggestion
        std::filesystem::path pathToLegacySettingsFile{ std::wstring_view{ pathToSettingsFile } };
```

if you find yourself using `c_str`, consider alternatives.